recipes/GFDL_SPEAR_SICDA/recipe.py

```diff
@@ -0,0 +1,54 @@
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+
+dates = [str(x) for x in range(1982,2018)]
+
+### Data set 1 ###
+
+def make_url1(time):
+    return "ftp://ftp.gfdl.noaa.gov/perm/William.Gregory/SIC_observations_NT/nt_"+time+"_v01_n_neareststod_spear.nc"
+
+time_concat_dim1 = ConcatDim("time", dates)
+pattern1 = FilePattern(make_url1, time_concat_dim1)
```

Comment:
```suggestion
pattern1 = FilePattern(make_url1, time_concat_dim1, file_type="netcdf3")
```
@William-gregory, based on the logs for the last failure, my guess is that the inputs for `sic-obs` are netcdf3?

Reply:
ah yes sorry, that's some lazy copy and pasting from me there. Hopefully that should be resolved now!